templates/user/edit-profile.html

```diff
@@ -288,11 +363,41 @@
                             {% else %}
                                 <a href="{{ url('disable_2fa') }}" class="button inline-button">{{ _('Disable') }}</a>
                             {% endif %}
+                            <span class="inline-header grayed">
+                                {{ _('Scratch Codes:') }}
+                                {% if profile.scratch_codes %}
+                                    <a id="generate-scratch-codes-button" href="{{ url('generate_scratch_codes') }}"
+                                        class="button inline-button">{{ _('Regenerate') }}</a>
+                                {% else %}
+                                    <a id="generate-scratch-codes-button" href="{{ url('generate_scratch_codes') }}"
+                                        class="button inline-button">{{ _('Generate') }}</a>
+                                {% endif %}
+                                <div class="hidden-word">{{ _('Hidden') }}</div>
```

Comment:
```suggestion
                                <div id="hidden-word">{{ _('Hidden') }}</div>
```

there is only a single one of these on the page